ee/tabby-webserver/src/service/cron/mod.rs

```diff
@@ -0,0 +1,7 @@
+mod db;
+mod job;
+mod run_async;
+mod run_sync;
+
+pub use run_async::run_offline_job_async;
+pub use run_sync::run_offline_job;
```

Comment:
I do not understand why there's a need to split run_offline_job_async / run_offline_job_sync.

I would suggest just expose a single function `run_cron`